update-lists.py

```diff
@@ -134,6 +135,7 @@ async def fetch_and_save_url(url, output_dir):
             aiohttp.ClientResponseError,
             aiohttp.client_exceptions.ClientConnectorError,
         ) as e:
```

Comment:
shouldn't we capture+log ALL exceptions? incl related to file creation ans writing

Reply:
we shouldn't really be having other exceptions, but I believe those will be handled by GitHub Actions reporting (since they'd be top-level uncaught exceptions).

Reply:
But will we get a Slack alert for that? Just wondering what would happen in the case where file writes weren't succeeding (for instance because the disk was full).